include/seqan/parallel/parallel_tags.h

```diff
@@ -62,15 +63,149 @@ namespace seqan {
  * @tag ParallelismTags#Serial
  * @headerfile <seqan/parallel.h>
  * @brief Tag to select the serial implementation of an algorithm.
+ *
+ * @tag ParallelismTags#Vectorial
+ * @headerfile <seqan/parallel.h>
+ * @brief Tag to select the vectorized implementation of an algorithm.
  */
 
 struct Parallel_;
 typedef Tag<Parallel_> Parallel;
 
+// ----------------------------------------------------------------------------
+// Tag Vectorial
+// ----------------------------------------------------------------------------
+
+struct Vectorial_;
+using Vectorial = Tag<Vectorial_>;
+
+// ----------------------------------------------------------------------------
+// Class ExecutionPolicy
+// ----------------------------------------------------------------------------
+
+/*!
+ * @class ExecutionPolicy
+ * @headerfile <seqan/parallel.h>
+ * @brief Policy to select runtime execution mode for algorithms.
+ * @signature template<typename TThreadingMode, typename TVectorizationMode>
+ *            struct ExecutionPolicy;
+ * @tparam TThreadingMode Type specifying the threading model.
+ *         Can be @link ParallelismTags#Parallel @endlink or @link ParallelismTags#Serial @endlink (default).
+ * @tparam TVectorizationMode Type specifying the vectorization model.
+ *         Can be @link ParallelismTags#Vectorial @endlink or @link ParallelismTags#Serial @endlink (default).
+ *
+ * The <tt>ExecutionPolicy</tt> class is used to select different execution models for certain algorithms.
+ * Depending on the specialization of the template parameters 4 different modes are possible: sequential, parallel,
+ * vectorized and parallel+vectorized. The number of threads for the parallel execution modes can be configured via a
+ * member variable.
+ */
+// Dynamic execution policy.
+template <typename TThreadModel = Serial, typename TVectorSpec = Serial>
+struct ExecutionPolicy
+{
+    // Member variables.
+    /*!
+     * @var size_t ExecutionPolicy::numThreads
+     * @brief The number of threads to use for the parallel execution. Defaults to 1.
+     */
+    size_t mNumThreads = 1;
```

Comment:
since when do members in SeqAn get a `m` prefix, why not just `_` like all the rest?